code/style.css

```diff
@@ -2,6 +2,236 @@
+}
+
+/* .hero-image-text {
+  font-size: 20px;
+  text-align: center;
+} */
+ 
+
+.hero-image-text {
+  position: absolute;
+  text-align: center;
+  top: 42%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-family: 'Lobster', cursive;
+  Font-Size: 30px;
+  color:  #48463c;
+  background: rgb(225,225,225, 0.7);
+  border: 1px solid  #0b6ba8;
+  padding: 20px;
+}  
+
+
+h2 {
+  font-family: 'Lobster', cursive;
+  Font-Size: 32px;
+  margin: 10px;
+  color:#48463c;
+}
+
+label {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-size: 16px;
+}
+/**** FLEX PARENT ****/
+
+.flex-parent {
+  display: flex;
+  height: 800px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  margin: 15px;
+  
+}
+
+.flex-child {
+  width: 345px;
+  border: 5px solid #0b6ba8;
+  border-radius: 5px;
+  
+}
+
+/**** ACCORDION ****/
+
+.accordion {
+  background: lightblue;
+  
+}
+
+.accordion-element {
+  border-top: 1px solid #0b6ba8;
+  padding:0px
+}
+
+
+.accordion-header {
+  background: #fff;
+  font-family: 'Roboto Condensed', sans-serif;
+  Font-Size: 22px;
+  margin: 0;
+  padding: 20px;
+}
+
+
+.accordion-text-container {
+  height: 0;
+  overflow: hidden;
+  transition: height .3s ease;
 }
+
+.accordion-text-container.active {
+  height: 150px;
+}
+
+.accordion-text {
+  font-family: 'Roboto Condensed', sans-serif;
+  Font-Size: 16px;
+  margin: 0;
+  padding:15px;
+}
+
+.active {
+  height: auto;
+}
+
+
+/**** FORM ****/
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  padding: 25px;
+  justify-content: center;
+  height: 275px;
+}
+
+
+.form-group {
+  margin-bottom: 15px;
+  display: flex;
+  align-items:center;
+  height: 25px;
+}
+
+.form-group label {
+  width: 50%;
+
+}
+.form-group input {
+  width: 200px;
+  height: 30px;
+}
+
+
+
+
+/**** FOOTER ***/
+
+
+.footer {
+  display: flex;
+  background: #0b6ba8;
+  height: 60px;
+  margin: 0px;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  position: absolute;
+}
+
+.footer-text {
+  Font-Family: Helvetica;
+  Font-Size: 10px;
+  color: #fff;
+  text-align: center;
+  font-size: 15px;
+  padding: 20px;  
+} 
+
+
+/**** MEDIA QUERIES ****/
+
+@media (max-width: 600px) {
+  
+  .hero-image {
+    height: 380px;
+    background-image: url('balloons.jpg');
+    background-repeat: no-repeat;
+    background-size:cover;
+    position: relative;
+  }
+  
+  
+  .image3 {
+      width: 100%;
+  }
+
+  
+}
+
+@media (min-width: 901px) {
+  
+  .hero-image {
+    height: 380px;
+    background-image: url('balloonride-big.jpg');
+    background-repeat: no-repeat;
+    background-size:cover;
+    position: relative;
+  }
+  
+  .image3 {
+      width: auto;
+  }
+
+  .flex-parent {
+    display: flex;
+    height: 500px;
+    flex-direction: row;
+    align-items: space-between;
+    justify-content: space-evenly;
+    margin: 20px;
+    
+  }
+  
+
+  .hero-image-text {
+    Font-Size: 50px;
+    }
+  
+}
```

Comment:
Very good that you don't have to write that much code to have a fully responsive site - great job 💥